### DISCUSSION TITLE
App mode

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#65] [Ideas]
I need suggestions about the App mode feature.
the idea is to replace the old way of creating sections.

Sections will be renamed as Apps
when you create an app (aka section) you will be able to see only things related to this app (isolation)

is that a good feature to implement ?

**Comment by Giorgi625** [comment]
So each app will have own control panel? that sounds interesting but we need more info to understand fully what you mean.
So when we create one app and type for this app in type page we will not see other types from other apps sections right? 
I think thats good but what about template files, like there will be separate folder for templates for each app or as it was before all templates will be in single folders and I can use same template for different apps?

**Comment by andiamine** [comment]
Dashboard
------------------
Admin will see sections (Apps) as cards + empty card with label "add new app"
If admin click on app card new view will open where can see only this app related things:
- App name
- App toolbar (Types, categories, records ....)

Once app clicked will be active, if admin logged out will not still active.
Admin will be able to switch from an app to another true select option
Fullscreen mode will be added to user will not see full template (modules, logo template ...) will see only JoomCCK.

**Comment by andiamine** [comment]
Templates and types are shareable , mean can be used by multiple apps.
Maybe we will add a new feature, that let the type/template to be shareable or not.

**Comment by Giorgi625** [comment]
I think this will be great option, especially if admin has option to choose share or not template files as so other people who manage will not get confused in some cases. I support you in this decision.

**Comment by LordHelmchen666** [comment]
Sounds good.

**Comment by andiamine** [comment]
I will start this feature this week :) i think should be implemented.

I will add an option in config that let admin switch between classic mode or app mode.
and app mode will be experimental , i will improve it slowly until become the default one.
and also will let users to help by new ideas and suggestions to this new mode.

This doesn't mean i will not continue or give less time to the stability of JoomCCK , making JoomCCK stable is the main goal.
